packages/browser-runtime-core/src/interpreter/interpreter.spec.ts

```diff
@@ -71,15 +71,15 @@ describe('Interpreter', () => {
       ).to.deep.equal({ x: 1 });
     });
 
-    it.skip('evaluates object literal after other statements as block', async() => {
-      // TODO: for some reason this is the default behaviour in node repl and devtools,
-      // as is now it wraps everything, which breaks the code.
-      expect(
-        await testEvaluate(
-          ';{x: 1}'
-        )
-      ).to.equal(1);
-    });
+    // it('evaluates object literal after other statements as block', async() => {
```

Comment:
this is just a difference we have with the usual node repl or chrome devtools:

in node:

```
> ;{x: 1}
1
> 
```

in browser-repl:

```
;{x: 1}
undefined
```

Reply:
Oh interesting. It's treating `{}` as a block scope and not as an object...which would explain why if you just put in `{x: 1}` then it will give you a syntax error. We could make a ticket for it but I don't think it should be a blocker.

Reply:
Here's the ticket: https://jira.mongodb.org/browse/MONGOSH-256

I will keep the tests as `skip` so they will warn as a reminder TODO.